packages/contracts/src/OffchainResolver.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./IExtendedResolver.sol";
+import "./SignatureVerifier.sol";
+
+/**
+ * Implements an ENS resolver that directs all queries to a CCIP read gateway.
+ * Callers must implement EIP 3668 and ENSIP 10.
+ */
+contract OffchainResolver is IExtendedResolver, IERC165, Ownable {
+    string public url;
+    mapping(address => bool) public signers;
+
+    event NewSigners(address indexed signer, bool isSigner);
+    event UpdateUrl(string url);
+
+    error OffchainLookup(
+        address sender,
+        string[] urls,
+        bytes callData,
+        bytes4 callbackFunction,
+        bytes extraData
+    );
+
+    constructor(string memory _url, address[] memory _signers) {
```

Comment:
where is the owner being set?

Reply:
Hi lucas. So, the contract OffchainResolver is Ownable. So, the owner will be the deployer. You can see it here: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol#L36

Reply:
I mean, isn't the constructor supposed to pass the owner to the Ownable inherited contract?

```suggestion
    constructor(string memory _url, address[] memory _signers) Ownable(msg.sender) {
```

Reply:
I also saw this code structure, but it is not necessary in our case. When I tested the offchain deployment here in a local script, I saw that the contract deployer is set as the owner. So the change you suggest would not be necessary as it is already done with the deploy.